front/src/containers/Profile/index.js

```diff
@@ -62,7 +62,7 @@ const ProfileScreen = (props) => {
             <SubTitle>Seguindo</SubTitle>
             <SubTitle>150</SubTitle>
           </ProfileInfoItem>
-        </ProfileInfoContent>
+        </ProfileInfoContent> */}
 
```

Comment:
Are these comments? If so, better take it off, right?

Reply:
Yeah, we app don't will have folowers for now